### DISCUSSION TITLE
Support for SSO/SAML with external IdP

### DISCUSSION THREAD

**Discussion by aplatoff** [hcengineering/platform#4771] [Architecture]
Hi everyone,

We're not experts in SSO/SAML, and since there is some interest, could you please share any thoughts or ideas on what should be supported? Are there any solutions that could be helpful, etc.? Essentially, we're seeking your assistance in defining the requirements.

Note: we're focusing on enterprise-level SSO/SAML here. Social Logins (Google, GitHub, etc.) for Huly will be implemented shortly.

**Comment by kkrushnyakov** [comment]
Hello! 
For in-house usage 1st thing to implement is LDAP authentication. That is what currently looks like a blocker for usage of Huly for my project. That should not be too hard to implement, IMHO.

**Comment by blz-ea** [comment]
Would love to see support for OpenID Connect.
Would allow reuse of existing self-hosted (e.g. Keycloak) or hosted (e.g. Auth0) auth infrastructure.

**Reply by uhlhosting** [reply]
this works fine...

**Reply by blz-ea** [reply]
Oh, nice I see that support was added in https://github.com/hcengineering/platform/pull/6654

**Comment by cmarshall10450** [comment]
I'd love to use this as an internal tool in my company but we'd need to have support for Microsoft Entra ID for SSO as well as user provisioning. OpenID Connect would work for us.

**Comment by mohaa7** [comment]
Guys, has there been any development for LDAP over the past two years? Is it now possible to connect Huly to Active Directory?
